### PR TITLE
.img-responsive should only be used on <img>s

### LINKED ISSUE #277: .img-responsive should only be used on <img>s

### PR DESCRIPTION
**Pull request by kkirsche**
[Fixes #277] `.img-responsive` should only be used on ``s

### CODE REVIEW

**Comment by kkirsche** [comment]
Same as #279, for whatever reason I'm getting errors when I try to add tests

File: src/bootlint.js [hunk 1]
```diff
@@ -938,6 +938,12 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`', formControlsOnWrongTypes);
         }
     });
+    addLinter("E045", function lintNavbarButtons($, reporter) {
+        var imgResponsiveNotOnImg = $('.img-responsive:not(img)').length;
+        if (imgResponsiveNotOnImg) {
+            reporter('`.img-responsive` should only be used on `<img>`s', this);
```
Comment (cvrebert) [inline src/bootlint.js]: Should be `imgResponsiveNotOnImg` instead of `this`. This is probably the cause of the test failures.

Comment (kkirsche) [inline src/bootlint.js]: Fixed. It was the lack of a jQuery object as the second argument of reporter which caused it.

**Comment by cvrebert** [comment]
Could you add a testcase asserting that correct usage of `.img-responsive` on an `` results in no warnings?

**Comment by kkirsche** [comment]
Course. Give me 5–10 min

**Comment by kkirsche** [comment]
@cvrebert added

**Comment by cvrebert** [comment]
Thanks!

**Comment by kkirsche** [comment]
My pleasure. Happy to help out @cvrebert. Thanks for the review and help finding that test issue

**Pull request merged**